dspace/config/dspace.cfg

```diff
@@ -1549,6 +1549,19 @@ solr-database-resync.cron = 0 15 2 * * ?
 # Default is 14 (i.e. processes that are two weeks or older will be deleted)
 # process-cleaner.days = 14
 
+#---------------------------------------------------------------#
+#----------------GOOGLE CAPTCHA CONFIGURATION-------------------#
+#---------------------------------------------------------------#
+# Enable CAPTCHA verification on ePerson registration
+
+registration.verification.enabled = false
```

Comment:
This setting likely should do something on the backend, otherwise it's very easy to bypass the reCaptcha by either using the REST API or creating a custom client.

Currently, this setting does nothing on the backend, so the backend behaves the same regardless of whether this is set to `false` or `true`.